src/network/networkd-network.c

```diff
@@ -40,6 +40,10 @@ static int network_load_one(Manager *manager, const char *filename) {
         _cleanup_network_free_ Network *network = NULL;
         _cleanup_fclose_ FILE *file = NULL;
         char *d;
+        _cleanup_free_ const char *dropin_dirname = NULL;
+        _cleanup_strv_free_erase_ char **dropin_dirs = NULL;
```

Comment:
why "erase"? that's only used for stuff that like passphrases and cryptographic keys...